charts/pega/charts/installer/templates/_pega-installer-job.tpl

```diff
@@ -93,6 +98,10 @@ spec:
             cpu: "{{ .root.Values.resources.limits.cpu }}"
             memory: "{{ .root.Values.resources.limits.memory }}"
         volumeMounts:
+{{- if and .root.Values.installerMountVolumeClaimName (eq .root.Values.upgrade.automaticResumeEnabled "true") }}
+        - name: {{ template "pegaInstallerMountVolume" }}
+          mountPath: "/opt/pega/installer"
```

Comment:
Would it make sense to use `/opt/pega/mount/installer` so that mount follow same pattern  see below for kit is mounted at `/opt/pega/mount/kit`?

Reply:
Addressed